dockstore-webservice/src/main/java/io/dockstore/webservice/resources/NotificationResource.java

```diff
@@ -0,0 +1,112 @@
+package io.dockstore.webservice.resources;
+
+import java.util.List;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import io.dockstore.webservice.CustomWebApplicationException;
+import io.dockstore.webservice.core.Notification;
+import io.dockstore.webservice.jdbi.NotificationDAO;
+import io.dropwizard.hibernate.UnitOfWork;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.Authorization;
+import org.apache.http.HttpStatus;
+import org.hibernate.SessionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static io.dockstore.webservice.Constants.JWT_SECURITY_DEFINITION_NAME;
+
+@Path("/curation")
+@Api("/curation")
+@Produces(MediaType.APPLICATION_JSON)
+public class NotificationResource {
+    private static final Logger LOG = LoggerFactory.getLogger(NotificationResource.class);
+
+    // interface between the endpoint and the database
+    private final NotificationDAO notificationDAO;
+
+    // constructor
+    public NotificationResource(SessionFactory sessionFactory) {
+        this.notificationDAO = new NotificationDAO(sessionFactory);
+    }
+
+    // get a notification by its id
+    @GET
+    @Path("/notifications/{id}")
+    @UnitOfWork
+    @ApiOperation(value = "Return the notification with given id", notes = "NO Authentication", responseContainer = "List", response = Notification.class)
```

Comment:
I personally think that "NO Authentication" is unnecessary.  The absence of the lock icon in the swagger-ui + the absence of bearer in the swagger.yaml should be self-explanatory.  Others should chime in.

Reply:
I agree I don't think it is necessary since there is the lock. I think historically we did this for some endpoints which is why it was done here.

Reply:
I'm half half.
I don't think it is necessary. 
That said, its kind of nice as an indicator that you truly wanted to make it public as opposed to simply forgetting/messing up. I just wish that there was an proper annotation for it.

Reply:
I just added it because it was used on a similar endpoint I was following as an example